docs/docs/cmd/spo/site/site-unarchive.mdx

```diff
@@ -0,0 +1,61 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo site unarchive
+
+Unarchives a site collection
+
+## Usage
+
+```sh
+m365 spo site unarchive [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --url <url>`
+: URL of the site collection.
+
+`-f, --force`
+: Don't prompt for confirmation.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be a Global or SharePoint administrator.
+
+:::
+
+:::warning
+
+If a site remains archived for more than seven days, the reactivation fee will be calculated based on the entire storage capacity of the site.
+
+:::
+
+## Examples
+
+Unarchive a specific site collection.
+
+```sh
+m365 spo site unarchive --url "https://contoso.sharepoint.com/sites/Marketing"
+```
+
+Unarchive a specific site collection without confirmation prompt.
+
+```sh
+m365 spo site unarchive --url "https://contoso.sharepoint.com/sites/Marketing" --force
+```
+
+## Response
+
+The command won't return a response on success.
+
+## More information
+
+Pricing model for Microsoft 365 Archive: [https://learn.microsoft.com/en-us/microsoft-365/archive/archive-pricing?view=o365-worldwide](https://learn.microsoft.com/en-us/microsoft-365/archive/archive-pricing?view=o365-worldwide)
```

Comment:
Let's remove the language in the URL, this will ensure that the user sees the page in his default system language.

```suggestion
Pricing model for Microsoft 365 Archive: [https://learn.microsoft.com/microsoft-365/archive/archive-pricing](https://learn.microsoft.com/microsoft-365/archive/archive-pricing)
```